Core/src/org/sleuthkit/autopsy/geolocation/MapWaypoint.java

```diff
@@ -64,6 +67,18 @@
     private static final Logger logger = Logger.getLogger(MapWaypoint.class.getName());
     private final static String HTML_PROP_FORMAT = "<b>%s: </b>%s<br>";
     static private final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssX", Locale.US);
+    
+    private static final Map<Integer, Color> artifactTypesToColors = new HashMap();
+    
+    static {
+        artifactTypesToColors.put(BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_BOOKMARK.getTypeID(), Color.BLUE);
+        artifactTypesToColors.put(BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_LAST_KNOWN_LOCATION.getTypeID(), Color.RED);
+        artifactTypesToColors.put(BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_ROUTE.getTypeID(), Color.YELLOW);
+        artifactTypesToColors.put(BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_SEARCH.getTypeID(), Color.GREEN);
+        artifactTypesToColors.put(BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_TRACK.getTypeID(), Color.ORANGE);
+        artifactTypesToColors.put(BlackboardArtifact.ARTIFACT_TYPE.TSK_GPS_TRACKPOINT.getTypeID(), Color.ORANGE);
```

Comment:
You are getting a warning for the use of the deprecated value, add the annotation to ignore the warning.

Reply:
The annotation isn't playing well with the static block apparently. I think it's a fair warning that will remind us to remove this deprecated artifact type once it's removed.

Reply:
@ethteck and @kellykelly3, we will likely never remove the deprecated artifact type. At least not until there is a TSK 5, something which I currently have no reason to expect will happen soon. If we can't figure out where to put the suppress deprecation warnings annotation, we will just have to live with the warning. It will be annoying and more or less permanent, but we can only do our best within the constraints we have. I think that we shouldn't let this issue hold up the PR.